opds_import.py

```diff
@@ -404,7 +407,9 @@ def extract_feed_data(self, feed, feed_url=None):
             metadata[internal_identifier.urn] = Metadata(**combined_meta)
 
             # form the CirculationData that would correspond to this Metadata
-            c_data_dict = m_data_dict.get('circulation')
+            c_circulation_dict = m_data_dict.get('circulation')
+            xml_circulation_dict = xml_data_dict.get('circulation', {})
+            c_data_dict = self.combine(c_circulation_dict, xml_circulation_dict)
```

Comment:
This is where we allow elementtree to define its own arguments to the CirculationData constructor. It's not tested in core but I have a test in my content branch.